### ISSUE TITLE
Facebook pixel typings don't match documentation

### ISSUE THREAD

**Issue by IlyaSemenov** [stafyniaksacha/vite-plugin-radar#1]
In the version published at npm, the typings are:

```ts
declare type FacebookPixelOption = string | FacebookPixel;
declare type FacebookPixel = {
    /**
     * Pixel tag
     */
    id: string;
};
```

which contradicts what you describe in README:

```ts
      // Facebook Pixel (multiple tag can be set with an array)
      pixel: [
        {
          id: 'XXXXXXX',
        }
      ],
```

(based on typings, arrays are not supported).

I believe you didn't actually publish the latest version?

**Comment by IlyaSemenov** [comment]
Yeah, the npm version is clearly outdated, it doesn't support Yandex metrika.

Unfortunately, installing with `yarn add --dev https://github.com/stafyniaksacha/vite-plugin-radar` doesn't work.

**Comment by stafyniaksacha** [comment]
Hu! Yes I forgot to build before publish, my bad, I'll fix that :)

**Comment by stafyniaksacha** [comment]
@IlyaSemenov I published the `v0.2.0` version with a pre-publish script! This should not happen again! Can you tell me if it is OK ?

**Comment by IlyaSemenov** [comment]
Yeah, it seems to be ok now. Thanks.
